### ISSUE TITLE
missing term: 'leaf emission rate'

### ISSUE THREAD

**Issue by leova** [Planteome/plant-trait-ontology#331]
I found no equivalent for CO_325:0000021 'leaf emission rate'. It is defined as 'number of new leaves emitted over a period of time'.

Is it ok to say CO_325:0000021 is a subclass of TO:0000241 'leaf number'?

**Comment by elizabetharnaud** [comment]
As a comment: this is typically a breeder's trait that is not recorded in the morpho-taxonomic descriptors used by genebanks.

**Comment by phismith** [comment]
No: this is not a child of leaf number, since it is a ratio (it is, in
effect, leaf number divided by time). We do not really have a mature
approach to such terms; does it make sense to introduce a higher level term
'growth rate'?
BS
